src/runtime/Loader.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from "react";
+
+export type LoaderProps = { errors: React.ReactNode[] };
+interface Props {
+  render: React.ComponentProps<any>;
+}
+export class Loader extends Component<Props> {
+  private static _LoaderContext = React.createContext<any>(undefined);
+
+  public static get Render() {
+    return Loader._LoaderContext.Consumer;
+  }
+
+  public render() {
+    const LoaderProvider = Loader._LoaderContext.Provider;
+    const loader = this.props.render;
+    return (
+      <LoaderProvider value={loader}>{this.props.children}</LoaderProvider>
```

Comment:
New-lines please ;) makes this way easier to read IMO

Reply:
Just found out this is done by prettier - Will find a way to solve this

Reply:
Looks like it's not possible with prettier https://github.com/prettier/prettier/issues/3101

Should we change the formatter? I really agree with you in the point that it's way easier to read. Maybe based on this comment: https://github.com/prettier/prettier/issues/3101#issuecomment-585293789 we should add this library? https://github.com/beautify-web/js-beautify 

Thoughts?

Reply:
Sure we can try that. I'm also a fan of just enforcing the styling ourselves and not relying on prettier. But I know that this is more tedious.